pipeline.yml

```diff
@@ -9,13 +9,21 @@ jobs:
     - get: common
       resource: common-ops-staging
       trigger: true
+    - get: terraform-state
+      resource: state-file-tooling
+      trigger: true
+    - get: pipeline-tasks
     - get: concourse-stemcell
       trigger: true
     - get: concourse-release
       trigger: true
     - get: garden-runc-release
       trigger: false
     - get: master-bosh-root-cert
+  - task: terraform-state-to-yaml
```

Comment:
Sounds good--then we upload the yaml back to the bucket where the state file lives?